gameplay/how-to.../mekanism-simple-fission-safety.md

```diff
@@ -49,4 +49,26 @@ And turn it on...
 
 My reactor would have exploded! But the safety breaker turned the reactor off. This setup should be able to handle very large heating rates.
 
+## A slightly more advanced safety
```

Comment:
`A slightly more advanced safety` -> `Slightly more advanced safety` is more natural sounding imo :)